src/vs/vscode.proposed.d.ts

```diff
@@ -1444,9 +1444,26 @@ declare module 'vscode' {
 		readonly uri: Uri;
 
 		/**
-		 * Event fired when there are no more references to the `CustomDocument`.
+		 * The file system path of the associated resource. Shorthand
+		 * notation for [CustomDocument.uri.fsPath](#CustomDocument.uri). Independent of the uri scheme.
 		 */
-		readonly onDidDispose: Event<void>;
+		readonly fileName: string;
```

Comment:
I wouldn't add that anymore. This is from times where we wanted to make it easy to use nodejs' fs but nowadays people should use `vscode.workspace.fs`.

Reply:
Ok, I'll remote it.

/cc @rebornix  I know you also have `fileName` in the `NotebookDocument` proposal